src/components/Drawer/mdc-temporary-drawer.vue

```diff
@@ -0,0 +1,114 @@
+<!-- thanks to petejohanson for this implementation -->
```

Comment:
regarding the location of the file, you want to change the src tree ?
`src/components/[feature_xxx]/mdc-xxxx_yyyy.vue`

good idea, we can align on MDC src tree names. but let's avoid capitals in the path though